src/excmds.ts

```diff
@@ -1634,11 +1634,30 @@ export function snow_mouse_mode() {
 export function pied_piper_mouse_mode() {
     toys.music()
 }
-
+/**
+ * Drawable variant of [[no_mouse_mode]]
+ * In this mode, you can use the mouse or a pen table to draw. To erase any drawings, use [[eraser_toggle]]
```

Comment:
```suggestion
 * In this mode, you can use the mouse or a digital stylus to draw. To switch to an eraser, use [[eraser_toggle]]
```

I'm not 100% sure what you meant by "pen table" : )